target/linux/generic/hack-6.12/931-fix-compilation-warnings.patch

```diff
@@ -0,0 +1,23 @@
+--- a/include/linux/irq.h
```

Comment:
https://github.com/openwrt/openwrt/blob/main/target/linux/generic/PATCHES:

```
ALL patches must be in a way that they are potentially upstreamable, meaning:

- they must contain a proper subject
- they must contain a proper commit message explaining what they change
- they must contain a valid Signed-off-by line
```

Also include the actual warnings/errors in the commit message. Without them, it is hard to understand whether the suggested fix is the right approach.